expression/builtin_compare_test.go

```diff
@@ -214,33 +214,40 @@ func (s *testEvaluatorSuite) TestIntervalFunc(c *C) {
 	}()
 
 	for _, t := range []struct {
-		args []types.Datum
-		ret  int64
+		args   []types.Datum
+		ret    int64
+		getErr bool
 	}{
-		{types.MakeDatums(nil, 1, 2), -1},
-		{types.MakeDatums(1, 2, 3), 0},
-		{types.MakeDatums(2, 1, 3), 1},
-		{types.MakeDatums(3, 1, 2), 2},
-		{types.MakeDatums(0, "b", "1", "2"), 1},
-		{types.MakeDatums("a", "b", "1", "2"), 1},
-		{types.MakeDatums(23, 1, 23, 23, 23, 30, 44, 200), 4},
```

Comment:
Why remove this?

Reply:
Not removed, a field was added to determine if an error was returned

Reply:
done.